src/nvim/strings.c

```diff
@@ -1542,7 +1542,6 @@ int vim_vsnprintf_typval(char *str, size_t str_m, const char *fmt, va_list ap_st
         p++;
         if (length_modifier == 'l' && *p == 'l') {
           // double l = long long
-          length_modifier = 'L';
```

Comment:
I don't think this should be removed, otherwise the code will be quite confusing.

Reply:
Unsure if clang-analyzer has an ignore directive, will check.

Reply:
This section is absolutely needed, for Windows.